packages/core/src/State.ts

```diff
@@ -43,6 +43,22 @@ export function isStateConfig<
   return 'value' in state && '_event' in state;
 }
 
+function getOutput<TContext, TEvent extends EventObject>(
```

Comment:
I'm not exactly sure if this should be computed here - but for the time being let's keep it here.